input/fsh/SD_AddressExtensions.fsh

```diff
@@ -1,7 +1,7 @@
 Extension: CityCode
 Id: CityCode
 Title: "City Code"
-Description: "City expressed as a numeric value.  As of the date of publication, in accordance with the NCHS Instruction Manual Part 8, Vital Records Geographic Classification, 2014 (https://www.cdc.gov/nchs/data/dvs/IMP8_2014.pdf). (Extension)"
```

Comment:
Looking at the autobuild, this did not actually resolve the broken links for city code and district code

Reply:
Looks good to me:  https://build.fhir.org/ig/HL7/vrdr/branches/STU2.2-preview1-edits/StructureDefinition-CityCode.html